.github/pull_request_template.md

```diff
@@ -1,69 +1,78 @@
+    own text at the top, a reviewer will update the checklist for you.
 
-Before you start, please read our [contribution guidelines](https://docs.esmvaltool.org/en/latest/community/introduction.html).
-To understand how we review and merge pull requests, have a look at our [review guidelines](https://docs.esmvaltool.org/en/latest/community/review.html).
+    While the checklist is intended to be filled in by the technical and scientific
+    reviewers, it is the responsibility of the author of the pull request to make
+    sure all items on it are properly implemented.
+-->
 
-<!---
-Please fill in the GitHub issue that is closed by this pull request, e.g. Closes #1903
---->
-Closes #issue_number
+## Description
 
+<!--
+    Describe the idea of your changes here to communicate why we should accept
+    this pull request and what problem it solves.
 
-* * *
+    Before you start, please read our [contribution guidelines](https://docs.esmvaltool.org/en/latest/community/introduction.html).
 
-**Checklist for technical review**
+    Please fill in the GitHub issue that is closed by this pull request, e.g. Closes #1903
+-->
+- Closes/Fixes: #issue_number
+- Link to documentation:
 
-- [ ] [Create an issue](https://github.com/ESMValGroup/ESMValTool/issues) to discuss what you are going to do, if you haven't done so already (and add the link at the bottom)
-- [ ] The pull request has a descriptive title that can be used as a one line summary in the [changelog](https://docs.esmvaltool.org/en/latest/changelog.html)
-- [ ] The code is composed of functions of no more than 50 lines and uses meaningful names for variables and follows the [style guide](https://docs.esmvaltool.org/en/latest/community/introduction.html#code-style)
-- [ ] [Documentation](https://docs.esmvaltool.org/en/latest/community/introduction.html#documentation) is available
-- [ ] Please use `yamllint` to check that your YAML files do not contain mistakes
-- [ ] (Only if really necessary) Add any additional dependencies needed for the diagnostic script to setup.py, esmvaltool/install/R/r_requirements.txt or esmvaltool/install/Julia/Project.toml (depending on the language of your script) and also to package/meta.yaml for conda dependencies (includes Python and others, but not R/Julia). Also check that the license of the dependency you want to add and any of its dependencies are compatible with [Apache2.0](https://github.com/ESMValGroup/ESMValTool/blob/master/LICENSE).
+* * *
 
-New or updated [recipe/diagnostic](https://docs.esmvaltool.org/en/latest/community/diagnostic.html):
+## Before you get started
 
-- [ ] Documentation for the recipe/diagnostic is available in the `doc/sphinx/source/recipes` folder and an entry has been added to `index.rst`
-- [ ] [Provenance information](https://docs.esmvaltool.org/en/latest/community/diagnostic.html#recording-provenance) has been added and no warnings related to provenance are generated when running the recipe
```

Comment:
The text about warnings has gone missing, could you add it to the checklist?

Reply:
Added